com.greyrook.siptop.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=siptop
+Comment=sipgate for Desktop
+GenericName=sipgate for Desktop
+Exec=/usr/bin/siptop.sh %U
+Icon=/usr/share/pixmaps/icon.png
+Type=Application
+StartupNotify=true
+Categories=GNOME;GTK;Network;InstantMessaging;
```

Comment:
Is it a GNOME or a GTK app? It seems to use electron.

Reply:
Fixed it